test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1

```diff
@@ -0,0 +1,372 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "Experimental Feature Basic Tests - Feature-Disabled" -tags "CI" {
+
+    BeforeAll {
+        $skipTest = $EnabledExperimentalFeatures.Contains('ExpTest.FeatureOne')
+
+        if ($skipTest) {
+            Write-Verbose "Test Suite Skipped. The test suite requires the experimental feature 'ExpTest.FeatureOne' to be disabled." -Verbose
+            $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+            $PSDefaultParameterValues["it:skip"] = $true
+        } else {
+            $TestModule = Join-Path $PSScriptRoot "assets" "ExpTest"
+            $AssemblyPath = Join-Path $TestModule "ExpTest.dll"
+            if (-not (Test-Path $AssemblyPath)) {
+                ## When using $SourcePath directly, 'Add-Type' fails in AppVeyor CI runs with an 'access denied' error.
+                ## It turns out Pester doesn't handle an exception like this from 'BeforeAll'. It causes the Pester to
```

Comment:
Did we open an issue in Pester repo or does one already exist?

Reply:
Didn't open an issue yet. Will do.

Reply:
Can you add link to bug in comment?  Don't want future tests following this pattern if it gets fixed in Pester.

Reply:
Will do after I open the issue.